content/20-exploring-the-cluster/22-rancher-applications/index.en.md

```diff
@@ -30,18 +30,18 @@ Here are all the relevant details for the **`Longhorn`** chart, like the Current
 
 ### Step 3 (Longhorn):
 
-Before we install Longhorn, we have to make sure we have the prerequistes installed on each node in the cluster. Longhorn requires **`nfs`** and **`iscsi`**. Longhorn provides **`kubectl`** deployment manifests to install each package.
+Before we install Longhorn, we have to make sure we have the prerequistes installed on each node in the cluster. Longhorn requires **`nfs`** and **`iscsi`** under the hood. Longhorn provides deployment scripts via **`kubectl`** to install the dependencies.
```

Comment:
can we be more specific? instead of "under the hood" maybe "to be installed on each node" or "to be installed on each worker node"

technically, Longhorn isn't providing deployment scripts via kubectl, it's providing them via GitHub and they're executed on each node by a manifest that's applied by a `kubectl` command. can we sharpen that sentence a bit?

Reply:
reading it again... agree it needs to be cleaned up. will update the phrasing and wording.